flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBKeyedStateBackend.java

```diff
@@ -557,6 +557,16 @@ private void createDB() throws IOException {
 		this.defaultColumnFamily = columnFamilyHandles.get(0);
 	}
 
+	private static class Utils {
+		static String getPathAsString(Path path) {
+			String pathString = path.getPath();
```

Comment:
I think a cleaner approach would by along the lines of `new File(path.toUri()).getAbsolutePath()`, the constructor should take care of transforming the URI to a system dependent string.